### DISCUSSION TITLE
Stationierungswerkzeug Konfiguration

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#26] [Q&A]
Wie wird das Stationierungswerkzeug konfiguriert?

**Comment by gis-eni** [comment]
Hier hat zu Parametrierung im WebGIS 4 eigentlich nichts geändert. Bei den Diensten gibt es den Punkt ``Stationierung``. Dort kannst du neue Stationierungsthemen anlegen. Darunter musst du dann wie gehabt das Linien- und das Punktthema angeben und dann noch festlegen, wie die beiden Verknüpft sind.
